### DISCUSSION TITLE
UltiMaker Cura 5.3.0-beta.1

### DISCUSSION THREAD

**Discussion by MariMakes** [Ultimaker/Cura#14637] [Releases]
<i>This release does not yet contain the new tree support implementation by @ThomasRahm.<a href="https://github.com/Ultimaker/Cura/releases/tag/5.3.0-alpha%2Bxmas"> You can download a special Cura version for that here</a>.</i> 

**Interlocking Structure Generation**
This new setting enables you to interlock multiple materials together, even if those materials are not compatible, contributed by @BagelOrb. Try it for yourself <a href="https://www.thingiverse.com/thing:5842529?utm_source=changelog&utm_medium=github&utm_campaign=53_beta">with this test model</a>.

**Brim per Material**
A brim can now be printed with multiple materials in the same model, contributed by @BagelOrb

**Updated Recommended Print Settings Menu**
The number of settings available in the recommended has expanded. The redesign will make it easier to use intent profiles or save custom profiles. 

**Other new features and improvements.**
- Improved the Zseam behavior so Hide Seam is generated less randomly.
- Fixed the order in which concentric top/bottom patterns are printed.
- Replaced mentions of Ultimaker by UltiMaker to correctly reflect the merger.
- Updated Load Project window with a new design.
- Added tooling to automatically generate translation files to reduce the chance of human error. 
- Added quality type to slice meta-data so it can be displayed in the Ultimaker Digital Factory.
- Added a new way to update profiles by adding a Save/Compare Settings Button in custom window. 
- Improved onboarding so it is easier to add your first cloud printer or a non-UltiMaker printer.
- Included options to download .MSI and .PKG for easier software distribution. 
- Smoothed some support generation, so support is generated less jittery.
- Reduced unnecessary support generation for small overhang regions, like textures.
- Improved support generation so the support doesn't fold around the model. 
- Reduced stringing for Lift Head setting by removing the move in XY.
- Added a Minimum Layer Time Temperature setting.
- Added settings for Support Interface Wall Line Count, Support Roof Line Count, and Support Floor Line Count.
- Added Brim Inside Avoid Margin setting, contributed by @BagelOrb
- Added prefix 'Printing' to post processing plugin script "DisplayFilenameAndLayerOnLCD" contributed by @brunohenriquy
- Added Enable, G-code Before, and G-code After To Filament Change, contributed by victornpb
- Added support for per-object retraction settings, contributed by @jeremysalwen
- Updated PauseAtHeight so M18 becomes M84 when disarming, contributed by @Cycov
- Added an option to keep stepper motors engaged to Pause At Height, contributed by @Cycov
- Added an option to Beep at Pause to Pause At Height, contributed by @Cycov
- Added Additional Gcode Fields to Pause At Height, contributed by @Cycov
- Make Layer Number the default in Pause At Height, contributed by @discip
- Added more information when rotating a model using the arrows, contributed by @Fabbro03
- Added the "CTRL +" shortcut to open Preferences window, contributed by @Ladnessness
- Added the option to add Machine Action buttons in the Printer Preferences, contributed by @julianCast

**Bug fixes:**
- Fixed a slicing crash if they were placed in the center of the buildplate. 
- Fixed a slicing crash that would happen if Adaptive Layers Maximum Variation had a negative value.
- Fixed a bug where an "object of type SettingFunction is not JSON serializable" error would prevent users from using certain plug-ins. 
- Fixed a bug where hole expansion would affect interlocking holes.
- Fixed a bug where fuzzy skin would also apply to interlocking structures. 
- Fixed a bug where Relative Extrusion was not calculated correctly when using Retract Continue.
- Fixed a bug that would prevent advanced users from slicing a batch with multiple STLs in a script.
- Fixed a bug where skirt lines would not follow the convex contours of models.
- Fixed a bug where Exclusing Slicing Tolerance was not applied correctly to bottom-layer(s).
- Fixed a bug where small travels were not retracted correctly. 
- Fixed a bug where the printjob time showed 0 minutes if adaptive layer height was enabled.
- Fixed a bug with Skin Overlap where the skin would overlap with the roofing.
- Fixed a bug where Support Interface would be generated on top of support.
- Fixed a bug where the Floor Support Interface would not be generated in certain models.
- Fixed a bug where a long material list would disappear from the screen.
- Fixed a bug where support is not generated correctly if Support Pattern is Concentric.
- Fixed a bug where an increased Skin Removal Width would cut off part of the skin area.
- Fixed a bug where Support Towers would be generated but would support nothing.
- Fixed a bug where Small Feature Size would also be applied to inner-wall loops.
- Fixed a bug where adhesion would be printed outside of the buildplate, contributed by @BagelOrb
- Fixed a bug where a primetower brim was printed when it shouldn't, contributed by @BagelOrb
- Fixed a bug where Ooze/Draft shields did not avoid prime tower, contributed by @BagelOrb
- Fixed a bug where not all Mac users could not use USB printing, contributed by Sovenger
- Fixed input value for platform offset when adding a logo, contributed by @drewdoggy
- Fixed a bug where the travel move at the end of a layer unretracts, contributed by @richfelker
- Fixed a bug where WallToolPaths are not simplified, contributed by @Piezoid
- Fixed a bug where iterators were moving out of range, contributed by @Piezoid and @plaintoothpaste
- Fixed a bug where the Maximum Area Deviation value was too small, contributed by @Piezoid
- Fixed a bug where an airgap was present in Spiralized Outer Contour, contributed by @richfelker
- Fixed a bug where speeds defined for slower layers would impact speeds accelerations and jerks for the whole print, contributed by @richfelker
- Fixed a bug where materials with longer names in Materials Preferences were harder to read, contributed by @Ghostkeeper  

**Printer definitions, profiles and materials:**
- Support Towers are working again and enabled by default for all printers.
- Updated Support Horizontal Expansion, Support Join Distance, and Minimum Support Area Settings for UltiMaker printers to improve support reliability. 
- Updated intent and quality files for UltiMaker printers to have more consistent file names. 
- Updated default Support Interface Pattern to ZigZag for UltiMaker printers
- Updated Maximum Combing Distance for UltiMaker printers
- Updated UltiMaker ABS Silver Metallic to the correct UltiMaker ABS Silver
- Updated AnkerMake M5 definitions, contributed by @just-trey
- Updated Anycubic i3 Mega S/Pro profile, contributed by @NilsRo
- Updated Creality Ender CR10 start Gcode, contributed by @InstantMuffin
- Updated Creality Sermoon V1 profiles to include hotend specifics, contributed by @Iggomez
- Updated Geetech A10M, A10, A20, A20M, A20T, A30M, MizarS, contributed by @whoseyoung
- Updated PPGF30 temperature from 235 to 240, introduced by @heed818
- Updated Rigid3D machine definitions, contributed by @mehmetsutas
- Updated Sovol SV01, SV02, and SV03, contributed by @eropple
- Updated Sovol's SV01 and SV02 bed dimensions, contributed by @eropple
- Added AnkerMake M5, contributed by @just-trey
- Added Anycubic Kobra Go, contributed by @Razzeee
- Added Artillery Hornet, contributed by @wilds
- Added Biqu Hurakan, contributed by @looxonline
- Added Blocks ONE MKII, Blocks One, Blocks Pro S100, Blocks Pro S30, Blocks R21, Blocks RD50, and Blocks Zero, contributed by @jgrilo-blocks
- Added Creality CR-10 Smart, contributed by @eropple
- Added Creality Ender-3 S1 Plus, and Ender-3 S1 Pro, contributed by @izilzty 
- Added Creality Sermoon V1, and Sermoon V1 Pro, contributed by @lggomez
- Added Dagoma Sigma, and updated Dagome Pro430, contributed by @0r31
- Added Elegoo Neptune 1, Neptune 2s, Neptune 3, Neptune 3 Max, Neptune 3 Plus, Neptune 3 Pro, and Neptune X, contributed by @NARUTOfzr
- Added FablabBcn PastePrinter, contributed by @EDUARDOCHAMORRO
- Added Flsun V400, contributed by @dengdaitianhei
- Added Fusion3 F410, contributed by @64bittuning
- Added Geetech A10Pro, A30Pro, A30T, E180, GiantArmD200, I3ProB, i3ProC, I3PrW, MeCreator, MeCreator2, MeDucer, Mizar, MizarM, MizarMax, MizarMax, MizarPro, and Thunder, contributed by @whoseyoung
- Added Klema 180, Klema 250, Klema 250 Pro, Klema 250 Twin, and Klema 500, contributed by @polyntsev
- Added Modix V3 BIG-120X, Modix V3 BIG-120Z, Modix V3 BIG-180X, Modix V3 BIG-40, Modix V3 BIG-60, and Modix V3 BIG-Meter, contributed by @Modix3D
- Added SnakeOil Standard 180, and SnakeOil Standard 250, contributed by @ChipCE
- Added Sovol SV01PRO, SV05, and SV06, contributed by @eropple
- Added UMO+ DXU, contributed by @ansonl
- Added Voron Trident 250, Trident 300, and Trident 350, contributed by @RagingRoosevelt
 
**Community translations:**
- Updated French translations, contributed by @xommit
- Updated a typo in Japanese translations, contributed by @tamorikawa
 
**Known critical issues:**
- Under certain angles, the Interlocking Structures are not effective, yet.
- The Gcode Reader Plug-in is broken since Cura 5.2, and will not load gcodes correctly yet. 
- Some models have been slicing slower since Cura 5.1. https://github.com/Ultimaker/Cura/issues/14108
- Reports have been coming in that Cura is running slow on Windows 11 https://github.com/Ultimaker/Cura/issues/13820 
- Multiple external monitors on Windows (especially if from the same brand) <a href="https://support.ultimaker.com/hc/en-us/articles/6103674944156">might be a problem under some circumstances</a>. 

_If you are looking for an AppImage for your Linux OS we recommend `UltiMaker-Cura-5.3.0-beta.1-linux.AppImage`. If you run into issues with launching your AppImage, we have `UltiMaker-Cura-5.3.0-beta.1-linux-modern.AppImage.asc` ready as an alternative._

Does Cura (not) work on your OS (version)? See <a href="https://support.ultimaker.com/hc/en-us/articles/360011889579">this article</a> for clarification.

<hr /><em>This discussion was created from the release <a href='https://github.com/Ultimaker/Cura/releases/tag/5.3.0-beta.1'>UltiMaker Cura 5.3.0-beta.1</a>.</em>

**Comment by gjosef** [comment]
Hello everybody, I am a retired Spanish engineer and an enthusiast of new technologies including 3d printing. I would like to contribute to the Spanish translation of the excellent program.

So if you see it convenient you can send me the texts, paragraphs, metascript, etc and I will translate them.

Greetings

**Reply by MariMakes** [reply]
Hey @gjosef,

We have translations available for Cura. You can find them here: https://github.com/Ultimaker/Cura/tree/main/resources/i18n
But there seems to be a mistake in this release, and they are not included.
We are working on a solution. Apologies for the inconvenience.

**Reply by jellespijker** [reply]
Hi @gjosef super kind offer indeed!

Fortunately it looks like we manage to fix the issue. The newly uploaded beta.2 binaries should contain the translations, please let us know if there is anything amiss with them or if you have suggestion how we can improve them.
 
![image](https://user-images.githubusercontent.com/8535734/220977851-b54dc72c-3559-4b3c-a3dd-fb64692be7c1.png)

**Reply by gjosef** [reply]
Hello to all,

I think I have expressed myself badly, I was referring not to the translation of the programme, which I have in Spanish, but rather to the Settings Guide plug-in, which I think can be of great help in Spanish as well.

thanks

**Reply by MariMakes** [reply]
Hey @gjosef,

That is wonderful news! The Settings Guide is not maintained by Ultimaker but by one of our lovely community members, Ghostkeeper. He is looking for a new maintainer. If you have considerable experience in 3D printing, minor technical know-how, and you want to keep updating this plug-in to reflect newly introduced settings and insights every minor Cura release, you can help contribute to a plug-in that has over 900,000 downloads. [You can volunteer or leave some appreciation for the plug-in here.](https://github.com/Ghostkeeper/SettingsGuide/issues/110)

**Comment by spraker1** [comment]
I need some help. I am new to 3d printing and I'm using chromebook. I downloaded the new update now how do I share it with linux? I had someone else set this up initially.  I need to know what commands to put in the terminal as well after sharing with linux.

**Reply by MariMakes** [reply]
Hey @spraker1,

Chromebooks are officially not supported, yet. 
But I've seen people helping each other to get it installed via this workaround: https://community.ultimaker.com/topic/27382-cura-on-chromebook-how/#comment-289389

Hope this helps!

**Comment by ghost** [comment]
As a new Anycubic Kobra Go owner, I am delighted to see a new profile for this printer. Thank you all, and thank you @razzeee!

**Comment by fieldOfView** [comment]
A prerelease of the Sidebar GUI that is compatible with Cura 5.3 is now available here:

https://github.com/fieldOfView/Cura-SidebarGUIPlugin/releases/tag/v4.5.1-DEV

Download the curapackage for 5.0 and newer and drop it onto the buildplate in Cura as if you were loading a 3d model.

**Comment by victornpb** [comment]
@MariMakes I know it's a bit silly but is there any reason why my name is the only one that doesn't have a proper mention [[1]](https://github.com/Ultimaker/Cura/releases#:~:text=contributed%20by%20victornpb)? 🥺
I know is a small contribution but you know, I feel quit proud about it 🥳

**Reply by jellespijker** [reply]
There is nothing silly about getting a mention. You should be very proud of it, no matter the size of a contribution you made, you helped us and the community, and that is really appreciated. 

I think we just made a boo-boo by not mentioning you. Sorry for that. Can you tell me what your contribution was so that I can update the release notes on the GitHub release, Then @MariMakes can update the changelog such that you're mentioned in the stable release.

**Reply by victornpb** [reply]
Just to be clear, my username (victornpb) is [there](https://github.com/Ultimaker/Cura/releases#:~:text=contributed%20by%20victornpb).
What I meant by silly is that the username isn't mentioned (mention as in `@` link).
This is the PR #13852

**Reply by jellespijker** [reply]
aha, `@` added in front of your username.

@MariMakes no action required for the `changelog.txt`

and thanks again for your contribution @victornpb

**Reply by MariMakes** [reply]
🙇 Apologies! 
There were so many changes this time around, it must have slipped through. 
Thanks for restoring it. 👍

**Reply by DataDivaNet** [reply]
Any chance the update is causing the platform to park pushed all the way forward instead of back during filament changes, and not retracting the filament before the nozzle cools?  Being able to add gcode will be handy, but right not it's harder to change the filament, and I have some projects with 8 different colors.

**Comment by DARoberto** [comment]
I'm planning to buy a Sovol SV04 printer. Will there be a profile in the near future for this machine? Thank you in advanced.

**Reply by jellespijker** [reply]
@eropple made a huge contribution for a lot of Sovol printers including this one. He did some quality work on that.

**Reply by jellespijker** [reply]
sorry I was mistaken he did a lot but not the SV04.

he probably did have access to one

**Comment by Modix3D** [comment]
A small edit to the notes: The Printer definitions for the Modix printers include the V4 big 60, V4 120x, V4 120z, V4 180x, and V4 Big meter

**Reply by MariMakes** [reply]
Hey @Modix3D,

You are totally correct! 
I'll update the release notes on Github, and I've updated the description for the stable release here https://github.com/Ultimaker/Cura/commit/b5b82224395179ca87c2b3d39b86a52566cd98b3

Thanks for contributing! 👑

**Comment by maruhe** [comment]
will your rename the "old" Tree-Support to "organic" and call the new one "Tree"? ;-P for real, i think that names just fit better like that
and could you also tell the guys from PusaSlicer how to do organic support?
the old Cura-Tree is the absolute best support!! only missing feature is a different layer heigt for Model and Support - for special when using adaptive layers
<3

**Reply by ThomasRahm** [reply]
What situation did you encounter where the new tree support is worse than the old one ?

**Reply by MariMakes** [reply]
Hey @maruhe,

If I understand correctly the New Tree Support will replace the old one.
I agree with @ThomasRahm, do you have an example of where you see a drop in quality? 
There is still time to fix it before it becomes stable. 

If you have feedback on how to make it better, we would love to hear more about it here: 
https://github.com/Ultimaker/Cura/discussions/14111

**Reply by maruhe** [reply]
at the begin of this year I saw PrusaSlice is adding organic support and after some weeks of waiting I was shocked when I saw the result the first time
when I saw your alpha I thought it was a joke to tease the prusa-guys ;-P
replacing the old support with that "xmas-tree" would be really strange - I don't see a reason to remove a great feature just because another feature was added

the main reason I admire the "old tree" is why it can fix it self - so I can print at high speed and when something goes wrong 10 layers after all is fine again
printing all this single round trees seems to be buch more dangerous on big objects and if 1-2 layer fail on one tree you may have to cancel the print
also the printer is louder on this single circles with fast travels to the next tree instead of one big oval move as the old support does
on the models I tested the new tree also creates support when I'm sure it's not possible to print it - I didn't try as it needs a day of printing to reach that point

I think the "new tree" (i really how you just rename the old one to "organic") is nice for some models and there is a lot of potential but it's not the best option for me
on my model it also seems the old is faster

I attached some screens (I really like the last where I created some x-mas-tree-like) and will also give some feedback on the other thread

![grafik](https://user-images.githubusercontent.com/49123648/221688518-52779f73-e615-4cce-8816-50b3fca29bba.png)
![grafik](https://user-images.githubusercontent.com/49123648/221688669-697786d0-c487-4a86-90e4-64fe9e72d669.png)

**Comment by Khronos666** [comment]
почему то не работает накат когда толщина стенки кратна меньше 3-х диаметров сопла....а расстояние очистки внешних стенок вообще не работает...
(GOOGL) for some reason, the knurling does not work when the wall thickness is a multiple of less than 3 nozzle diameters....and the cleaning distance of the outer walls does not work at all...
![Desktop Screenshot 2023 02 25 - 15 43 11 45](https://user-images.githubusercontent.com/123404675/221352978-06a6ba8a-2486-4524-a6f3-fd9a43e2d335.png)
![Desktop Screenshot 2023 02 25 - 15 42 47 55](https://user-images.githubusercontent.com/123404675/221352986-b799600e-3613-4c4f-8be2-2282c2611e52.png)

**Reply by MariMakes** [reply]
Hey @Khronos666,

Do you have a project file for us? It contains the printer and settings we need for troubleshooting. 
To save a project file go to File -> Save project. Please make sure to .zip your project file. For big files, you may need to use WeTransfer or similar file-sharing sites.

You'll need to open Github in your browser to attach any files to your ticket
You can click the **view it on GitHub** link at the bottom of your email to open GitHub in the browser.

**Reply by Khronos666** [reply]
Вот файлы проектов с alpha и beta версии...
кстати alpha схожа со всеми версиями 5.х.х...(перепробовал все версии 5.х.х, удалял все профили и настройки...принтер custom)
(googl) Here are the project files with alpha and beta versions ...
by the way, alpha is similar to all versions 5.x.x ... (I tried all versions 5.x.x, deleted all profiles and settings...custom printer)

[Cura 5.3.0-alpha_beta_.zip](https://github.com/Ultimaker/Cura/files/10847319/Cura.5.3.0-alpha_beta_.zip)

**Comment by mikedandan** [comment]
Constantly getting error when slicing with this version.  I have tried both in tree support and normal.  I keep having to use the older version.  I have a flsun v400 profile

**Reply by maruhe** [reply]
i have a lot of probles also since Cura 5.2..
check if the tree has enough space - f.e. when when you use Support Roof/Floor the distance from support-bottom to top needs to be high enough
so if your model is close to the build-plate there may just be to less space
..but sometime I also got errors like that I couldn't explain..

**Reply by MariMakes** [reply]
Hey @mikedandan,

Is it possible that your printjob is located at 0, 0, 0, and it slices if you move it a little away from the center of the buildplate? 

If that's not it, I'll need a little bit more information from you.
Do you have a project file for us? It contains the printer and settings we need for troubleshooting. 
To save a project file go to File -> Save project. Please make sure to .zip your project file. For big files, you may need to use WeTransfer or similar file-sharing sites.

You'll need to open Github in your browser to attach any files to your ticket
You can click the **view it on GitHub** link at the bottom of your email to open GitHub in the browser.

**Reply by maruhe** [reply]
oh, and rotating the model 180° also helped me sometimes

**Reply by MariMakes** [reply]
Hey @mikedandan,

Quick update, during the beta we made even more improvements to reduce the amount of slicing crashes. 
The 5.3 stable should be..... more stable 😅 

Thanks again for reporting

**Comment by ungutknut** [comment]
It looks like support isn't generated for thin structures with a steep angle. This used to work fine on 5.2.2 and 5.3.0 alpha (x-mas) and doesn't anymore on 5.3.0 beta2. No matter the settings, that bottom edge of the wings won't get supported. Attached you can find the project file.
[EN3_Vought_F4U_Corsair.zip](https://github.com/Ultimaker/Cura/files/10841499/EN3_Vought_F4U_Corsair.zip)
Since this isn't my discovery, here's the link to the original reddit thread:
https://www.reddit.com/r/Cura/comments/11dbkdy/supports_53_beta_vs_522/

**Reply by MariMakes** [reply]
Hey @ungutknut,

Thanks for the report! 
We've taken a look at this, but seem to have forgotten to update you. 

We made an effort to resolve it here https://github.com/Ultimaker/CuraEngine/commit/4e6304765306204a92b8456ea9398ffac23be0d6
This will be available in the Stable version of 5.3 👍

**Comment by goofy173** [comment]
This beta 2 says some of my printer configurations are corrupt. Specifically my Geeetech A10T printer of which I have 3 setups in cura. 5.2.1 works fine.

**Reply by maruhe** [reply]
me to

**Reply by MariMakes** [reply]
Hey! @goofy173 and @maruhe 

Thanks for your report. 🎉 
I think I figured out what went wrong! @5axes was also experiencing something similar. 

In an effort to add more Geetech printers, some have been removed to be replaced. 
https://github.com/Ultimaker/Cura/commit/36ac3d77440ccc2aa55db18326dfb51b3246b7df

We are working on a solution so other Geetech users won't encounter this issue in the stable release.  
This report is making other reports fall into place! Thanks ❤️

**Comment by SerialExperimentsPT** [comment]
Updating from 4.13.0: 
Why is it that the Ubuntu repositories are so far behind the current release?  Similarly, snap seems to be similarly behind at 4.13.2..

**Comment by chrisw31** [comment]
Hi, I see brim is now printing from the inside out, this is causing a problem for me, because with the Bowden setup I have, the filament needs more time to start extruding correctly, so I end up with a mess around the part, and the brim ends up not being attached to the part, which defeats the purpose. Thanks

**Reply by MariMakes** [reply]
Hey @chrisw31,

Thank you so much for reporting this!
You sparked quite some discussion in the team about the intended behavior. 
After which we agreed it was a mistake that slipped through. 

The outside in behavior has been restored, and can be used once the stable is released. 
https://github.com/Ultimaker/CuraEngine/commit/f66ed64457484f065f41f6c9d9047814bd490695

Thank you again for flagging this 💕

**Reply by chrisw31** [reply]
@MariMakes thanks very much for for listening, I'm glad everything was worked through productively :)

**Reply by MariMakes** [reply]
![ezgif-3-63f9ef79f3](https://user-images.githubusercontent.com/40423138/223436292-98146ca3-366a-48b7-878d-868ecd343c81.gif)
Sneakpeak 😉

**Reply by chrisw31** [reply]
@MariMakes Cool, I like the look of that!

**Comment by Dozer-hh** [comment]
Where has the "Save profile with changes" and "Save as new profile" dialog gone to?
(Using the german language version and don't know, how the dialog is truely called in english version.

![Cura_Profil_Dialog](https://user-images.githubusercontent.com/60253043/223112355-88c95230-d56c-4201-87e9-931e69467dbf.jpg)

**Reply by maruhe** [reply]
there should be a little save-icon next to the profile-name 
may it becomes visible when you change something
and you can edit profiles in general settings also

**Reply by MariMakes** [reply]
![image](https://user-images.githubusercontent.com/40423138/223181891-0ff99abc-7209-436e-a9e1-913aaa471619.png)

@maruhe is correct. 
We've added this release a new save icon, to make it easier for people to update their profiles. 
You can also just click it to see which changes you made that have not been saved yet.

**Reply by Dozer-hh** [reply]
Yes, but with german language "Save" Button leads to a non-intuitive dialog. English ist ok, but in germaan it says "Änderungen speichern" or "Benutzerdefiniertes Profil". If you click "Benutzerdefniertes Profil" the name is empty. In stable version, the file name is preset to the existing name.

**Reply by maruhe** [reply]
german translation is also confusing when you change profile and have some settings changed
in english the button text is "keep changes" while in german the used word means "save" changes instead of "keep"
"Änderungen behalten" would be correct german and less confusing

**Reply by Dozer-hh** [reply]
Ok, so this was not fixed for the final 5.3.0.

**Reply by MariMakes** [reply]
I found the translation strings you are referring to. 
https://github.com/Ultimaker/Cura/blob/d031451909160c048269dddf38c97be2e992ec4f/resources/i18n/de_DE/cura.po#L4602

Do I understand correctly that one of these two strings need to be changed?
What do they need to be changed into? 

msgid "Save Custom Profile"
msgstr "Benutzerdefiniertes Profil"

msgid "New Custom Profile"
msgstr "Benutzerdefiniertes Profil"

**Reply by Dozer-hh** [reply]
Yes, that's correct, the german strings / butttons need to be changed.
And one functionallity is missing at new profiles.

Seee following screenshots...


![Change_Dialog_1](https://user-images.githubusercontent.com/60253043/224070242-090bee96-570c-4107-8b7b-bc42df237f3c.jpg)

![Change_Dialog_2](https://user-images.githubusercontent.com/60253043/224070208-01dee33c-b352-468d-89cb-10ef4173055f.jpg)

**Reply by maruhe** [reply]
this also needs to be changed:
#: resources/qml/Dialogs/DiscardOrKeepProfileChangesDialog.qml:164
msgctxt "@action:button"
msgid "Keep changes"
msgstr "Änderungen speichern"

#: resources/qml/Dialogs/DiscardOrKeepProfileChangesDialog.qml:178
msgctxt "@action:button"
msgid "Save changes"
msgstr "Änderungen behalten"

correct would be 
"Keep changes" = "Änderungen behalten"
and 
"Save changes" = "Änderungen speichern"

**Reply by discip** [reply]
@Dozer-hh & @maruhe
Please have a look: #14836

**Comment by Czspecter** [comment]
Hi everyone,
 I don't know if I'm at the right place but, I have an issue if slicing a particular model type. It is a sinusoidal wall structure inspired by a cuttlefish bone. I would like to print the one line walls with a 0.2 mm nozzle. However in CURA 5.2.0 (and 4.3.0), it has problem of slicing it even after some .stl repair. Each model was made in Solidworks and the walls were conceived with a 0.2, 0.25, 0.28 and 0.4 mm thickness respectively. I tried to print a simple test wall, modeled in Solidworks as well and find out that the minimal wall thickness to be detected by CURA for a 0.2 nozzle is 0.25 mm. However it doesn't work for my model. Therefore my question is : What is the minimal wall thickness of a model in CURA to slice it properly?

**Reply by MariMakes** [reply]
In Cura 5.x we have the following settings that might be worth tuning:

- Minimum Wall Line Width
- Print Thin Walls
- Minimum Feature Size
- Minimum Thin Wall Line Width

Hope this helps 😇 
These are the type of questions that usually get more insightful answers on our community forum here:  https://community.ultimaker.com/

**Reply by Czspecter** [reply]
Hi,

Unfortunately it still didn't slice the model properly, but thank you for your reply. I will try to ask the community.

Have a nice day.

**Comment by Dozer-hh** [comment]
Since Cura 5.xx there is a slicing Problem with objects, that are as high as first layer hight.
For perfect first layer calibration I print an object with 0.2 mm layer hight and an area of 100 x 100 or 200 x 200 mm.
(The area is irrelevant for that!)

Since Cura 5.x you can't slice that object, while in Cura 4.x this is not a problem.
In Cura 5.x the object has to be a little bit higher (f.e. 0.21mm) than the first layer hight.

I am wondering, that no one seems to have reported that before.

![Slicing_Problem_FirstLayer-Test_1](https://user-images.githubusercontent.com/60253043/224252636-a93bbdb2-89f0-45cf-98c4-51123649885f.jpg)

![Slicing_Problem_FirstLayer-Test_2](https://user-images.githubusercontent.com/60253043/224252668-ee8c5627-7c40-4d5a-aace-c18418d6a2bd.jpg)

**Reply by Dozer-hh** [reply]
I circled the wrong entry on the right side of the screenshots (layer hight instead of first layer hight), but result is the same, because the values are 0.2 mm on both entrys.

**Reply by Dozer-hh** [reply]
@MariMakes Is this a known problem?

**Reply by MariMakes** [reply]
Hey @Dozer-hh,

This is one of the first times I've seen this. I wouldn't be surprised if this is related to the variable line width engine. 
I'll check with the team what they think.  👍

**Reply by Dozer-hh** [reply]
@MariMakes Any news on this?

**Reply by MariMakes** [reply]
Hey @Dozer-hh,

So sorry, didn't have a chance to properly follow up. 
I'm having some trouble reproducing the issue on my side on Cura 5.3.1.
![image](https://github.com/Ultimaker/Cura/assets/40423138/8a264cfd-5de8-456f-b1cd-3283c87bd70d)

Could it be related to your settings? 

Do you have a project file for us? It contains the printer and settings we need for troubleshooting. 
To save a project file go to File -> Save project. Please make sure to .zip your project file. For big files, you may need to use WeTransfer or similar file-sharing sites.

You'll need to open Github in your browser to attach any files to your ticket
You can click the **view it on GitHub** link at the bottom of your email to open GitHub in the browser.

**Reply by Dozer-hh** [reply]
Hey @MariMakes 
here is a file with the described problem.

[First-Layer_200x200x0.2mm_VMPLAHH20230411_230521.zip](https://github.com/Ultimaker/Cura/files/11527673/First-Layer_200x200x0.2mm_VMPLAHH20230411_230521.zip)

**Reply by MariMakes** [reply]
Hey @Dozer-hh ,

I think I figured out what is causing your slice to fail.
It's the slicing tolerance!  If you change it to middle it seems to be resolving your issue. That's also why I had trouble reproducing it. 

![image](https://github.com/Ultimaker/Cura/assets/40423138/ffe6c5f5-ffdf-49ff-b3d5-4f7a0b2adcc1)

**Reply by Dozer-hh** [reply]
@MariMakes 
Thank you for finding that solution!

**Comment by tyhowell13** [comment]
Hi Y'all,

I'm pretty new to the 3D printer realm & just got my own Neptune 3pro. I'm currently using the Elegoo Cura V4.8 slicing software that came out of the box with the printer. I noticed there was an update for Ultimaker Cura 5.3. I followed the download link & it looks like there are quite a few files to download. Being the newb I am, currently using a windows PC, I'm assuming I should download the Ultimaker-Cura-5.3.0-win64.exe. Second, is it recommended to also download the "Source Code" file / what is the source code file for? 

Thanks in Adv!

**Reply by MariMakes** [reply]
Hey @tyhowell13,

Welcome to the 3D printer realm 🚀 

Elegoo Cura is a fork of Cura based on a version of Ultimaker Cura of September 2020, created by the Elegoo team. 
UltiMaker Cura is maintained by the Ultimaker team and her community members.
Since January 2023, we have a printer profile of Elegoo Neptune 3pro available. 

![image](https://user-images.githubusercontent.com/40423138/224715813-8863e592-5dd2-4367-925f-791b270a593e.png)

Since we support windows, mac and linux devices, there are a number of options to choose from. 

If you are just a personal windows user the .exe should be perfectly fine. 
[UltiMaker-Cura-5.3.0-win64.exe](https://github.com/Ultimaker/Cura/releases/download/5.3.0/UltiMaker-Cura-5.3.0-win64.exe)

The MSI is more of a best practices in companies and schools because it's easier to install on multiple computers at once.
[UltiMaker-Cura-5.3.0-win64.msi](https://github.com/Ultimaker/Cura/releases/download/5.3.0/UltiMaker-Cura-5.3.0-win64.msi)

Hope this helps you to get unstuck. 💪 
Good luck and happy printing!

**Comment by afish0381** [comment]
I'm not sure if this is the proper location for this, sorry if it is.  I've noticed that smart hiding seams, looks to be not very different than random seams since the new update.  Is this my imagination or is something going on with this setting?  Thanks for your time
Arthur

**Reply by MariMakes** [reply]
Hey @afish0381,

We've made some improvements in the Zseam this release. 
If you want to make full use of it you have to change your preference from Smart Hiding to None. 
Can you let us know if that works for you?

**Comment by Raven-XIII** [comment]
Hey all not sure where to drop this but I've got an easy workaround for the current Beta issue on Cura 5.4.0-Beta. 

Simple Workaround For Missing Material Profiles.

It's pain in the butt but temporary fix/workaround. Open Cura 5.3.1 & export profiles needed then manual import to Beta. Like I said a pain but a lot quicker than a txt editor.

Works without issue and you can selectively import only what you need ATM.


Issue https://github.com/Ultimaker/Cura/issues/15771
